bin/parse_go_sum

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+
+require 'json'
+
+parsed_file = nil
+begin
+  parsed_file = File.read(ARGV[0])
+rescue StandardError
+  warn 'Unable to read and parse go.sum file'
```

Comment:
Is it possible to write a unit-test for this warning?

[Here](https://github.com/coinbase/salus/blob/aad632b77a5fbd20af302f3f08dd53d8dba6e2ea/spec/lib/salus/scanners/report_podfile_lock_spec.rb#L77) is an example.

Reply:
Yup, will add

Reply:
Actually, might not be needed to rescue here since 'empty' and 'missing' go.sum file is being handled by the calling class, so should be safe to remove from here. Will update accordingly.